tgui/packages/tgui/interfaces/MatMarket.js

```diff
@@ -0,0 +1,113 @@
+import { useBackend } from '../backend';
+import { Section, Stack, Flex, Button } from '../components';
+import { Window } from '../layouts';
+
```

Comment:
```suggestion

`type Data = {
    orderingPrive: BooleanLike; // you will need to import this
    canOrderCargo: BooleanLike;
    creditBalance: number;
    materials: Material[];
};

type Material = {
    name: string;
    quantity: number;
    id: string; // correct this if its a number
    trend: string;
};
    
```

when you're done change the extension to tsx
.js files should also be .jsx but the bundler accepts the difference